### ISSUE TITLE
Add parser for /sys/block/<device>/queue

### ISSUE THREAD

**Issue by SuperQ** [prometheus/procfs#266]
https://www.kernel.org/doc/Documentation/block/queue-sysfs.txt

There are a number of useful files in this subsystem.

Related to https://github.com/prometheus/node_exporter/issues/1601

**Comment by binjip978** [comment]
Would like to help with this one and have two questions:
It will be ok if we will return queue stats for particular device, like this?
```go
func (fs FS) SysBlockDeviceQueueStats(device string) (BlockQueueStats, error) {}
```
And for optional fields like `throttle_sample_time` it's ok to use pointer in BlockQueueStats structure?

**Comment by pgier** [comment]
I believe this is resolved with the merge of #299
